pkg/adapters/goframe/go.mod

```diff
@@ -0,0 +1,54 @@
+module github.com/alibaba/sentinel-golang/pkg/adapters/goframe
+
+go 1.20
```

Comment:
plz downgrade the go version to no higher than 1.18 to keep it consistent with main package, so as to avoid introducing new features now or in the future and support more go versions